bakkesmod-playlist-rank-viewer/pch.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
What does this file do? What does pch mean?

Reply:
pch is a precompiled header in c++. I'm not usually a c++ programmer but I've seen people use them as sort of the primary include, intended to prevent multi-line copy/pastes I believe. I felt this would help make it a bit easier to grow without cluttering up the component header too much. We can always do something different if you prefer.

Reply:
I actually think this file might be overkill looking at it again. I think when I added this I intended to separate a bit more of the code but it didn't end up needing all that. If it suits you I might revert this change.

Reply:
Heh im not a C++ programmer so Ill accept your judgement blindly on this :)